internal/testrunner/runners/system/servicedeployer/compose.go

```diff
@@ -178,6 +178,12 @@ func (s *dockerComposeDeployedService) TearDown() error {
 	}); err != nil {
 		return errors.Wrap(err, "could not shut down service using Docker Compose")
 	}
+
+	// Remove the outputs generated by the service container
+	if err := os.RemoveAll("/tmp/" + s.ctxt.Test.RunID); err != nil {
```

Comment:
I suggest moving this into the `defer` block up near L160 so that this is cleaned up even on error.